### PR TITLE
Add types for cytoscape-euler

### PR DESCRIPTION
**Pull request by rogerdubbs**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [X] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [X] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [X] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## cytoscape-euler ([<kbd>unpkg</kbd>](https://unpkg.com/browse/cytoscape-euler@latest/))
was missing the following properties:
1. cytoscapeEuler
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9ddad0c84fd14b202fab05fe251a78831eaeb853
</p>

File: types/cytoscape-euler/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
+// Type definitions for cytoscape-euler 1.2
+// Project: https://github.com/cytoscape/cytoscape.js-euler
+// Definitions by: Roger Dubbs <https://github.com/RogerDubbs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import cytoscape = require('cytoscape');
+
+declare const cytoscapeEuler: cytoscape.Ext;
+export = cytoscapeEuler;
```
Comment (peterblazejewicz) [inline types/cytoscape-euler/index.d.ts:9]:
```suggestion
export = cytoscapeEuler;
export as namespace cytoscapeEuler;
```
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
(check the published package)
https://unpkg.com/browse/cytoscape-euler@1.2.2/webpack.config.js

Comment (rogerdubbs) [reply types/cytoscape-euler/index.d.ts:9]: Thanks for the suggestion - I incorporated it.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@rogerdubbs thanks!

**Comment by rogerdubbs** [comment]
Ready to merge

**Pull request merged**